etc/ci-gather-coverage.sh

```diff
@@ -82,3 +82,6 @@ if [[ -f gap-coveralls.json ]]
 then
   curl -F json_file=@gap-coveralls.json "https://coveralls.io/api/v1/jobs"
 fi
+
+# upload C/C++ coverage to coveralls.io
+coveralls --follow-symlinks -n -e pkg/ -e bin/ -e etc/ -e extern/ -e gen/ -e tst/
```

Comment:
we'll also need to exclude `julia*/` for the Julia builds. Well, or I guess we could try to place the Julia binary in some other directory to hide it, like so (untested):
```patch
diff --git a/etc/ci-prepare.sh b/etc/ci-prepare.sh
index 605444ebdf..15d92ae2a1 100644
--- a/etc/ci-prepare.sh
+++ b/etc/ci-prepare.sh
@@ -30,11 +30,12 @@ fi
 if [[ $JULIA = yes ]]
 then
   # TODO: once Julia 1.1 is released, switch to stable Julia versions here?
+  pushd extern
   wget https://julialangnightlies-s3.julialang.org/bin/linux/x64/1.1/julia-b55b85cd99-linux64.tar.gz
   #wget https://julialangnightlies-s3.julialang.org/bin/linux/x64/julia-latest-linux64.tar.gz
   tar xvf julia-*-linux64.tar.gz
   rm julia-*-linux64.tar.gz
-  pushd julia-*
+  cd julia-*
   JULIA_PATH=$(pwd)
   popd
   CONFIGFLAGS="--with-gc=julia --with-julia=$JULIA_PATH $CONFIGFLAGS"
```